src/components/Pod.js

```diff
@@ -1,120 +1,129 @@
 import React, {Component} from 'react';
-import styled from 'styled-components';
+import {TimelineMax, Power1} from 'gsap';
+import GSAP from 'react-gsap-enhancer';
 
 import {isCenter, getDirection, DIRECTIONS} from '../station';
-import {times, sizes, zIndex, animations} from '../constants';
-
+import {sizes} from '../constants';
 import pod from '../img/pod.svg';
+import hub from '../img/hub';
+
+const getAnimationProps = (direction, isOriginStation) => {
+  let curve, z, scaleBegin, scaleEnd, west = false;
+
+  if(isOriginStation) {
+    scaleBegin = 1;
+
+    switch(direction) {
+      case DIRECTIONS.northwest:
+        [curve, z, scaleEnd, west] = [hub.paths.back_left.bezier, 2, 0.33, true];
+        break;
+      case DIRECTIONS.northeast:
+        [curve, z, scaleEnd] = [[...hub.paths.back_right.bezier].reverse(), 2, 0.33];
+        break;
+      case DIRECTIONS.southwest:
+        [curve, z, scaleEnd, west] = [[...hub.paths.front_left.bezier].reverse(), 2, 2, true];
```

Comment:
I like this notation more because I don't have to write () around it every time. I don't think it makes a difference anyway.